bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingFactoryHelper.java

```diff
@@ -144,8 +143,27 @@ private static Channel createChannel(ChannelDefinition channelDefinition, ThingU
             channelBuilder = channelBuilder.withDescription(channelDefinition.getDescription());
         }
 
+        channelBuilder = channelBuilder.withProperties(channelDefinition.getProperties());
+
+        Channel channel = channelBuilder.build();
```

Comment:
IIRC the JVM could do better optimization if you don't create a temporary (useless) variable.